daffodil-runtime1/src/main/scala/edu/illinois/ncsa/daffodil/processors/parsers/PrimParser.scala

```diff
@@ -1,72 +1,81 @@
-/* Copyright (c) 2012-2014 Tresys Technology, LLC. All rights reserved.
```

Comment:
Merged back into Parser. Allows us to seal base traits/abstract classes forcing derivation from either Prim, PrimNoData, or Combinator flavors. Prevents erroneous derivation of new traits/classes from the wrong parents.